cli/src/commands/subgraph/commands/check.ts

```diff
@@ -59,10 +69,13 @@ export default (opts: BaseCommandOptions) => {
       }
     }
 
+    // submit an empty schema in case of a delete check
+    const schema = schemaFile ? await readFile(schemaFile) : Buffer.from('type Query');
```

Comment:
Will this really empty be then? It could be possible that this definition makes the federated graph valid because at least one root query is required. @Aenimus

Reply:
It must be populated to be a valid federated graph.

E.g.,
```
type Query {
    dummy: String!
}```